src/components/renderers/layouts/masterdetail/masterdetail-renderer.ts

```diff
@@ -38,6 +42,9 @@ class MasterDetailRenderer implements JSONForms.IRenderer {
     }
 }
 
-angular.module('jsonforms.renderers.layouts.masterdetail').run(['RenderService', (RenderService) => {
-    RenderService.register(new MasterDetailRenderer());
-}]);
+export default angular
```

Comment:
Registering the module happens in `masterdetail-renderer.ts`, so I think this ok.